ddtrace/api.py

```diff
@@ -8,6 +8,7 @@
 
 
 log = logging.getLogger(__name__)
+log.setLevel(logging.WARNING)
```

Comment:
Not sure it's needed for all modules. When a logger is created, the default level is ``NOTSET`` that means everything is logged if the logger is the root logger. If the logger is not the root logger (i.e. ``ddtrace.tracer`` module), then it should search the level of the ancestor (``ddtrace``) and so on.

If all loggers have the same logging level, it's enough to set the level of the root logger:
```python
# in the ddtrace/__init__.py
log = logging.getLogger('ddtrace')  # you can use the __name__ it's just to be clear
log.setLevel(logging.WARNING)
```

Reference: https://docs.python.org/2/library/logging.html#logging.Logger.setLevel